File: app/src/main/java/com/bkjcb/rqapplication/fragment/ChooseApplianceCheckTypeFragment.java
package com.bkjcb.rqapplication.fragment;

/**
 * Created by DengShuai on 2019/12/30.
 * Description :
 */
public class ChooseApplianceCheckTypeFragment extends ChooseCheckTypeFragment {
    public static ChooseApplianceCheckTypeFragment newInstance(OnChooseListener listener, String[] type) {
        ChooseApplianceCheckTypeFragment fragment = new ChooseApplianceCheckTypeFragment();
        fragment.setListener(listener);
        fragment.setTypeString(type);
        return fragment;
    }



}

File: app/src/main/java/com/bkjcb/rqapplication/check/fragment/ChooseApplianceCheckTypeFragment.kt
package com.bkjcb.rqapplication.check.fragment

/**
 * Created by DengShuai on 2019/12/30.
 * Description :
 */
class ChooseApplianceCheckTypeFragment : ChooseCheckTypeFragment() {
    fun newInstance(listener: OnChooseListener?, type: Array<String>): ChooseApplianceCheckTypeFragment {
        val fragment = ChooseApplianceCheckTypeFragment()
        fragment.setListener(listener)
        fragment.typeString = type
        return fragment
    }
}